src/App/MainPage.xaml.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+using Microsoft.UI.Xaml.Controls;

```

Comment:
do you need both this and the one on line 19?

Reply:
No I do not! It's been removed